<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Kopfrechentrainer</class>
 <widget class="QMainWindow" name="Kopfrechentrainer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>263</width>
    <height>243</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kopfrechentrainer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="LblAufgabe">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>(Aufgabe)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="EditEingabe">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="CmdPruefenNeu">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Pr√ºfen / Neu</string>
    </property>
   </widget>
   <widget class="QLabel" name="LblKommentar">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>(Kommentar)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CmdEnde">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Ende</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>263</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
